endpoints/openrtb2/amp_auction_test.go

```diff
@@ -1028,40 +1028,50 @@ func TestAMPSiteExt(t *testing.T) {
 
 // TestBadRequests makes sure we return 400's on bad requests.
 func TestAmpBadRequests(t *testing.T) {
-	dir := "sample-requests/invalid-whole"
+	dir := "sample-requests/amp/invalid-whole"
```

Comment:
Tests `sample-requests/invalid-whole/**.json` are supposed to run too. I agree with your comments in the PR description above. In order to not to add 49 JSON files to `sample-requests/invalid-whole/amp/invalid-whole` that are going to be almost identical to their counterparts in `sample-requests/invalid-whole`, maybe we can filter out the `"request.app must not exist in AMP"` errors for now:
```
1030   func TestAmpBadRequests(t *testing.T) {
1031 -     dir := "sample-requests/amp/invalid-whole"
     +     dir := "sample-requests/invalid-whole"
1032       files, err := os.ReadDir(dir)
1033       assert.NoError(t, err, "Failed to read folder: %s", dir)
1034  
1035       for index, file := range files {
1036           filename := file.Name()
1037 -         fileData := readFile(t, "sample-requests/amp/invalid-whole/"+filename)
     +         fileData := readFile(t, "sample-requests/invalid-whole/"+filename)
1038  
1039 *-- 32 lines: test, err := parseTestData(fileData, filename)-------------------------
1071  
1072           response := recorder.Body.String()
     +         if strings.Contains(response, "request.app must not exist in AMP") {
     +             continue
     +         }
1073           assert.Equal(t, test.ExpectedReturnCode, recorder.Code, filename)
1074           assert.Contains(t, response, test.ExpectedErrorMessage, "Actual: %s \nExpected: %s. Filename: %s \n", response, test.ExpectedErrorMessage, filename)
1075       }
1076   }
endpoints/openrtb2/amp_auction_test.go
```
In a future PR we can either modify the 30 files in `sample-requests/invalid-whole` that come with an `"app"` field so they come with `"site"` instead whenever they are not testing any `"app"`-exclusive feature. Or find another workaround.